PowerEditor/src/Parameters.cpp

```diff
@@ -7163,7 +7165,10 @@ void NppParameters::createXmlTreeFromGUIParams()
 		TiXmlElement *GUIConfigElement = (newGUIRoot->InsertEndChild(TiXmlElement(TEXT("GUIConfig"))))->ToElement();
 		GUIConfigElement->SetAttribute(TEXT("name"), TEXT("openSaveDir"));
 		GUIConfigElement->SetAttribute(TEXT("value"), _nppGUI._openSaveDir);
-		GUIConfigElement->SetAttribute(TEXT("defaultDirPath"), _nppGUI._defaultDir);

```

Comment:
Restore this line.
Add `_lastUsedDir` and use this variable for memorizing instead of using other variables.